sdks/java/io/hadoop/jdk1.8-tests/build.gradle

```diff
@@ -69,3 +75,16 @@ task packageTests(type: Jar) {
 }
 
 artifacts.archives packageTests
+
+// The cassandra.yaml file currently assumes "target/..." exists.
+// TODO: Update cassandra.yaml to inject new properties representing
+// the root path. Also migrate cassandra.yaml to use any open ports
+// instead of a static port.
+task createTargetDirectoryForCassandra() {
+  doLast {
+    if (!project.file("target").exists()) {
```

Comment:
Does it matter if this folder has left over files if it already exists?

Reply:
No. Its just a place for cassandra to dump content into while running an embedded instance.

Note the number of test failures decreased by 4 with this change. Only 3 failures in hbase.